src/components/datalist-item/DatalistItem.js

```diff
@@ -46,14 +47,14 @@ class DatalistItem extends React.Component<Props> {
     id: string;
 
     render() {
-        const { children, className, isActive, ...rest } = this.props;
+        const { children, className, isActive, title, ...rest } = this.props;
```

Comment:
title already part of rest btw which in turn is part of itemprops

Reply:
It looks like when I take out title from props, I'm getting an error saying that title is not defined:
<img width="1192" alt="Screen Shot 2019-06-03 at 10 36 07 AM" src="https://user-images.githubusercontent.com/7213887/58822068-764bd200-85eb-11e9-8f16-af71f6c794d9.png">